doc/nitrocli.1

```diff
@@ -12,11 +12,13 @@ It can be used to access the encrypted volume, the one-time password generator,
 and the password safe.
 .SS Device selection
 Per default, \fBnitrocli\fR connects to any attached Nitrokey device.
-You can use the \fB\-\-model\fR and \fB\-\-serial-number\fR options to select
-the device to connect to.
+You can use the \fB\-\-model\fR, \fB\-\-serial-number\fR and \fB\-\-usb-path\fR
+options to select the device to connect to.
 \fBnitrocli\fR fails if more than one attached Nitrokey device matches this
 filter or if multiple Nitrokey devices are attached and none of the filter
 options is set.
+Use the \fBlist\fR command to list all attached devices with their USB path,
```

Comment:
Yes!  This reminds me that I forgot to address the naming of the option in the PR description:

The path value originates in hidapi as the `path` field of the `hid_device_info` struct and is described as the “Platform-specific device path”.  In libnitrokey, it’s the `path` field of the `NK_device_info` struct.  In nitrokey-rs, it’s the `path` field of the `DeviceInfo` struct.

I did not want to use `--path` as the option name because it is very generic and I would expect a file system path as its value.  `--device-path` could have been an option, but then I would expect something like `/dev/hidraw0`.  At least on my machine, the device path is a combination of the USB bus, the USB device and (probably) the HID device index, e. g. `004:001:00`.  Therefore I decided to use the term USB path although, as far as I know, no-one else is using it.  What do you think?

Reply:
I'd say it's fine. It's explained pretty well in the man page and I share your concerns about "device path" being ambiguous, too (and probably more so). So as long as it's consistent within the program I think it's okay.